src/script/page/preferences/OptionPreferences.tsx

```diff
@@ -0,0 +1,274 @@
+    properties: {settings},
+  } = propertiesRepository;
+  const [optionAudio, setOptionAudio] = useState<AudioPreference>(settings.sound.alerts);
+  const [optionReplaceInlineEmoji, setOptionReplaceInlineEmoji] = useState<boolean>(settings.emoji.replace_inline);
+  const [optionDarkMode, setOptionDarkMode] = useState<boolean>(settings.interface.theme === ThemeViewModelThemes.DARK);
+  const [optionSendPreviews, setOptionSendPreviews] = useState<boolean>(settings.previews.send);
+  const [optionNotifications, setOptionNotifications] = useState<NotificationPreference>(settings.notifications);
+
+  useEffect(() => {
+    const updateProperties = ({settings}: WebappProperties): void => {
+      setOptionAudio(settings.sound.alerts);
+      setOptionReplaceInlineEmoji(settings.emoji.replace_inline);
+      setOptionDarkMode(settings.interface.theme === ThemeViewModelThemes.DARK);
+      setOptionSendPreviews(settings.previews.send);
+      setOptionNotifications(settings.notifications);
+    };
+    const updateDarkMode = (newDarkMode: boolean) => setOptionDarkMode(newDarkMode);
+
+    amplify.subscribe(WebAppEvents.PROPERTIES.UPDATE.INTERFACE.USE_DARK_MODE, updateDarkMode);
+    amplify.subscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
+
+    return () => {
+      amplify.unsubscribe(WebAppEvents.PROPERTIES.UPDATE.INTERFACE.USE_DARK_MODE, updateDarkMode);
+      amplify.unsubscribe(WebAppEvents.PROPERTIES.UPDATED, updateProperties);
+    };
+  }, []);
+
+  const saveOptionAudioPreference = (audioPreference: AudioPreference) => {
+    propertiesRepository.savePreference(PROPERTIES_TYPE.SOUND_ALERTS, audioPreference);
+    setOptionAudio(audioPreference);
+  };
+
+  const saveOptionEmojiPreference = (emojiPreference: boolean) => {
+    propertiesRepository.savePreference(PROPERTIES_TYPE.EMOJI.REPLACE_INLINE, emojiPreference);
+    setOptionReplaceInlineEmoji(emojiPreference);
+  };
+
+  const saveOptionNotificationsPreference = (notificationsPreference: NotificationPreference) => {
+    propertiesRepository.savePreference(PROPERTIES_TYPE.NOTIFICATIONS, notificationsPreference);
+    setOptionNotifications(notificationsPreference);
+  };
+
+  const saveOptionSendPreviewsPreference = (sendPreviewsPreference: boolean) => {
+    propertiesRepository.savePreference(PROPERTIES_TYPE.PREVIEWS.SEND, sendPreviewsPreference);
+    setOptionSendPreviews(sendPreviewsPreference);
+  };
+
+  const saveOptionNewTheme = (useDarkMode: boolean) => {
+    const newTheme = useDarkMode ? ThemeViewModelThemes.DARK : ThemeViewModelThemes.DEFAULT;
+    propertiesRepository.savePreference(PROPERTIES_TYPE.INTERFACE.THEME, newTheme);
+    setOptionDarkMode(useDarkMode);
+  };
+
+  return (
+    <div style={{display: 'flex', flexDirection: 'column', height: '100vh'}}>
+      <div className="preferences-titlebar">{t('preferencesOptions')}</div>
+      <div className="preferences-content" ref={setScrollbarRef}>
+        <section className="preferences-section">
```

Comment:
There is a component for the sections (PreferencesSection) that you can use, it might need a little bit of adjustments though.